easybuild/easyconfigs/p/ParaView/ParaView-5.2.0-foss-2017b-mpi.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.2.0'
+versionsuffix = '-mpi'
+
+homepage = "http://www.paraview.org"
+description = "ParaView is a scientific parallel visualizer."
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s.tar.gz"]
+
+dependencies = [
+    ('X11', '20170314'),
+    ('Mesa', '17.2.4'),
+    ('libGLU', '9.0.0'),
+    ('Qt', '4.8.7'),
+    ('zlib', '1.2.11'),
+    ('HDF5', '1.8.19'),
+    ('matplotlib', '2.1.0', '-Python-2.7.14'),
+]
+
+builddependencies = [('CMake', '3.9.4'), ('Python', '2.7.14')]
+
+checksums = ['894e42ef8475bb49e4e7e64f4ee2c37c714facd18bfbb1d6de7f69676b062c96']  # ParaView-v5.2.0.tar.gz
+
+patches = ['ParaView-PythonSix-Update.patch']
+separate_build_dir = True
+
+configopts = '-DPARAVIEW_INSTALL_DEVELOPMENT_FILES=ON -DPARAVIEW_USE_MPI=ON '
+configopts += '-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include -DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.%s ' % SHLIB_EXT
+configopts += '-DOSMESA_INCLUDE_DIR=$EBROOTMESA/include -DOSMESA_LIBRARY=$EBROOTMESA/lib/libOSMesa.%s ' % SHLIB_EXT
+configopts += '-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s ' % SHLIB_EXT
+configopts += '-DVTK_USE_SYSTEM_HDF5=ON -DPARAVIEW_ENABLE_PYTHON=ON '
+configopts += '-DVTK_WRAP_PYTHON=ON -DPARAVIEW_ENABLE_MATPLOTLIB=ON '
+configopts += '-DPARAVIEW_QT_VERSION=4 -DQT_QMAKE_EXECUTABLE=$EBROOTQT/bin/qmake '
+# if you want to build server only Paraview, uncomment the following line:
+# configopts += '-DVTK_USE_X=OFF '
+
+# Without internet connection turn off testing (uncomment the following line)
+# configopts += '-DBUILD_TESTING=OFF '
```

Comment:
we use this off by default, leads to too much trouble (tries to pull in stuff from the web)